<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadarPlayerMainWindow</class>
 <widget class="QMainWindow" name="RadarPlayerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Radar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="CarTableView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>640</y>
      <width>961</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ConsoleText">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>640</y>
      <width>801</width>
      <height>391</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="VideoLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>960</width>
      <height>540</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="MapLabel">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>180</y>
      <width>700</width>
      <height>375</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>1721</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="NextButton">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>下一帧</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PauseButton">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>190</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QLabel" name="fpsLabel">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
